pkg/virt-operator/kubevirt.go

```diff
@@ -1151,3 +1152,32 @@ func (c *KubeVirtController) syncDeletion(kv *v1.KubeVirt) error {
 	logger.Info("Processed deletion for this round")
 	return nil
 }
+
+// checkIfConfigMapStillExists This function validates that the obsolete kubevirt-config configMap is no longer exist.
+// The user should not use this configMap to configure KubeVirt, and KubeVirt will ignore it anyway. In case the configMap
+// still exists, this function emit an event to notify the user, and ask them to delete it.
+func (c *KubeVirtController) checkIfConfigMapStillExists(logger *log.FilteredLogger, trigger <-chan time.Time) {
+	ctx := context.Background()
+	getOpts := metav1.GetOptions{}
+
+	msg := fmt.Sprintf("the %s configMap is still deployed. KubeVirt does not support this configMap and it can be safely removed", obsoleteCmName)
+
+	go func() {
+		for range trigger {
+			logger.V(5).Info("read the kubevirt-config configMap. if exist, emmit an event")
+			cm, err := c.clientset.CoreV1().ConfigMaps(c.operatorNamespace).Get(ctx, obsoleteCmName, getOpts)
+			if err != nil {
+				if errors.IsNotFound(err) {
+					logger.V(5).Info("The obsolete kubevirt-config configMap could not be found. good.")
+				} else {
+					logger.Errorf("can't get the kubevirt-config configMap %v", err)
+				}
+			} else {
+				if cm != nil {
+					logger.Warning("The obsolete kubevirt-config configMap still exists. Please remove it.")
+					c.recorder.Eventf(cm, k8sv1.EventTypeNormal, obsoleteCMReason, msg)
```

Comment:
This should defintively be a `EventTypeWarning`.

Reply:
done